.conda/build.sh

```diff
@@ -1,13 +1,5 @@
 #!/bin/bash
 
-if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
Why did this block go away?

Reply:
`MACOSX_DEPLOYMENT_TARGET` is set by `conda-build` based on the SDK we pass. The compiler variables are not used; instead, `conda-build` installs and uses compilers based on its configuration.